jobs/jobs.json

```diff
@@ -194,6 +194,30 @@
         "testbench": "tb_idma_nd_midend",
         "synth_top": "idma_nd_midend_synth"
     },
+    "rw_axi_rw_axis": {
+        "jobs" : {
+            "simple"          : "backend_rw_axi/simple.txt"
```

Comment:
@chaoqun-liang, can you maybe add some better test cases here? This is just a placeholder for now.